examples/wheel/wheel_test.py

```diff
@@ -380,24 +380,30 @@ def test_rule_sets_stamped_version_in_wheel_metadata(self):
             "rules_python",
             "examples",
             "wheel",
-            "example_minimal_library-0.1._BUILD_TIMESTAMP_-py3-none-any.whl",
+            "example_minimal_library_BUILD_USER_-0.1._BUILD_TIMESTAMP_-py3-none-any.whl",
```

Comment:
Please update the test name, too, to better reflect what it's testing.: "test_rule_expands_workspace_status_keys_in_wheel_metadata" (or something to get across its not just the version attribute we're testing)

Reply:
Thanks, updated now.